composer.json

```diff
@@ -18,7 +18,7 @@
        }
```

Comment:
Since you add return types, you should also update the part of the _composer.json_ file where the PHP version is defined, because you drop the support for PHP 5.x which is also mentionned in the README.md  :)

Reply:
Sure thing, thanks for the review :)

Reply:
Hi, Eric. I have updated the PR, dropping the support for php 5.x. Now, the minimum is 7.x, while the latest version of php and dev requirement libs are also supported for the most up-to-date users. Please let me know if there's something else I can do. Thanks again.

Reply:
@danielgsims is your package stil maintained :) ?

Reply:
Since it is not marked as abandoned, can we consider merging the PR regardlessly?

Reply:
Well I am not a maintainer of this repo, so... :/